### PR TITLE
Correct manually `RSpec/RepeatedExampleGroupDescription` RuboCop offenses

### PR DESCRIPTION
**Pull request by eduardoj**

### CODE REVIEW

File: src/api/spec/controllers/webui/package_controller_spec.rb [hunk 1]
```diff
@@ -589,7 +589,7 @@ def remove_file_post
       login(user)
     end
 
-    context 'when triggering a rebuild fails' do
+    context 'non existant repository' do
```
Comment (krauselukas) [inline src/api/spec/controllers/webui/package_controller_spec.rb:592]:
```suggestion
    context 'non existent repository' do
```

Comment (krauselukas) [reply src/api/spec/controllers/webui/package_controller_spec.rb:592]: this is misspelled in other places of this context as well, maybe you can correct those as well :)

**Comment by krauselukas** [comment]
Rest looks good to me

File: src/api/spec/controllers/webui/package_controller_spec.rb (hunk 1 above)
Comment (eduardoj) [reply src/api/spec/controllers/webui/package_controller_spec.rb:592]: That's not the purpose of this pull request, sorry. That can be addressed in a following pull request.

Comment (eduardoj) [reply src/api/spec/controllers/webui/package_controller_spec.rb]: I applied your suggestion. This misspelling was introduced in this pull request. Please take a look again.

The rest of the misspellings could be addressed later on in other pull requests.

**Review by krauselukas — APPROVED** [review]

**Pull request merged**